packages/bits/src/lib/select-v2/selected-items-key-control.service.ts

```diff
@@ -1,4 +1,4 @@
-import { ActiveDescendantKeyManager } from "@angular/cdk/a11y";
+import { ActiveDescendantKeyManager, LiveAnnouncer } from "@angular/cdk/a11y";
```

Comment:
It might be good to add some unit tests for the changes in this file.

Reply:
add spec file for this service.